specs/phase0/p2p-interface.md

```diff
@@ -561,10 +561,9 @@ The request MUST be encoded according to the encoding strategy.
 The requester MUST close the write side of the stream once it finishes writing the request message.
 At this point, the stream will be half-closed.
 
-The requester MUST wait a maximum of `TTFB_TIMEOUT` for the first response byte to arrive (time to first byte—or TTFB—timeout).
-On that happening, the requester allows a further `RESP_TIMEOUT` for each subsequent `response_chunk` received.
+The requester MUST NOT make more than `MAX_CONCURRENT_REQUESTS` concurrent requests with the same protocol ID.
 
-If any of these timeouts fire, the requester SHOULD reset the stream and deem the req/resp operation to have failed.
+If a timeout happens on the requesting side, they SHOULD reset the stream.
```

Comment:
I'd suggest keeping the language consistent
`If a timeout occurs, the requester SHOULD reset the stream.`

Reply:
[a232b25](https://github.com/ethereum/consensus-specs/pull/3767/commits/a232b25e7f1067b1301be37c9daab9b8685ee4dc)